nekoyume/Assets/_Scripts/Game/Game.cs

```diff
@@ -589,7 +589,7 @@ private async void UploadLog(string logString, string stackTrace, LogType type)
             else
             {
                 const string groupName = "9c-player-logs";
-                var streamName = Agent.Address.ToString();
+                var streamName = _options.AwsSinkGuid;
```

Comment:
Because `aws-sink-guid` option is not required (i.e. optional), the streamName seems able to be `null`. Is it okay?